django_test_app/rest_app/views.py

```diff
@@ -121,3 +195,62 @@ class AsyncParseHeadersController(
 ):
     async def post(self) -> _CustomHeaders:
         return self.parsed_headers
+
+
+@final
+@ensure_csrf_cookie_json
+class CsrfTokenController(Controller[PydanticSerializer]):
+    """Controller to obtain CSRF token."""
```

Comment:
```suggestion
    """Controller to obtain CSRF token."""
    
    # We don't add `ensure_csrf_cookie_json.responses`,
    # because it always returns ones we already have
```